doc/source/uploading.rst

```diff
@@ -2,43 +2,40 @@ Uploading to pypi (or custom servers)
 =======================================
 
 When the (full)release command tries to upload your package to a pypi server,
-zest.releaser basically just executes the command ``python setup.py register
-sdist --formats=zip upload``.
+zest.releaser basically just executes the command ``twine upload``.
```

Comment:
Ah, true, `setup.py sdist` is still needed yes. I have updated the text along those lines.